src/display/annotation_layer.js

```diff
@@ -228,6 +228,10 @@ class AnnotationElement {
       container.setAttribute("aria-haspopup", "dialog");
     }
 
+    if (this.data.alternativeText) {
+      container.title = this.data.alternativeText;
```

Comment:
With this code being moved here, it can be shortened slightly.
```suggestion
    if (data.alternativeText) {
      container.title = data.alternativeText;
```